tests/kernel/install_ltp.pm

```diff
@@ -338,6 +338,11 @@ sub run {
         my $version = get_var('VERSION');
         assert_script_run("grub2-set-default 'SLES ${version}, with Xen hypervisor'");
         my $serial_console = get_serial_console;
+        # If there is no XEN replace grub entry on default installation we need to add empty one
```

Comment:
This reads strange. Could you please replace with the below one:
"If the default installation does not create a Xen boot entry, we must add an empty one."

Reply:
Indeed wrong.  It is option, not entry.